config/locales/en.yml

```diff
@@ -326,3 +326,11 @@ en:
               min_grade:          "must be greater than 0"
             user:
               blank:              "can't be blank'"
+        assignment:
+          attributes:
+            user:
+              blank:                 "User can't be null"
+            offspring:
+              blank:                 "must exist"
```

Comment:
If you don't validate this, it will not appear... anyway, I've never found that message..